### ISSUE TITLE
Make this a module for Mina

### ISSUE THREAD

**Issue by sandstrom** [moll/mina-rsync#12]
Would it make sense to try to get this merged into Mina as a plugin? Git is on a module-level in Mina, seems like this would be a good fit.

It would also increase the number of users and thus (likely) the number of contributors too.

**Comment by jalcine** [comment]
Yeah, since there's no activity (in terms of feedback from the maintainer or even active merges) for this project.
